crates/pop-contracts/src/test.rs

```diff
@@ -31,26 +41,60 @@ pub fn test_e2e_smart_contract(path: &Option<PathBuf>) -> Result<(), Error> {
 	Ok(())
 }
 
-#[cfg(feature = "unit_contract")]
 #[cfg(test)]
 mod tests {
 	use super::*;
-	use std::fs;
 	use tempfile;
 
-	fn setup_test_environment() -> Result<tempfile::TempDir, Error> {
+	#[test]
+	fn test_contract_test() -> Result<(), Error> {
```

Comment:
Personal opinion, but prefixing the test names with `test_` when there is a `#[test]` attribute seems redundant. I know it makes sense here due to the name of the function being tested, but I have seen it being used elsewhere. But then there is a suffix of `_test` - perhaps consider `_works` instead.

Reply:
I removed the `_test` suffix and renamed the tests included in this PR. 
I agree it makes sense for the other tests to exclude the `test_` prefix, which will be addressed in a separate PR.